docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -699,3 +699,9 @@ Temporal recommends rotating API keys periodically.
 
 A: The full key is displayed only once upon creation for security reasons.
 If you lose it, generate a new one.
+
+**Q: What does the `Generate API Key` button on a Namespace page do?**
+
+A: The `Generate API Key` button on a Namespace page generates an API key with permissions for the given Namespace.
```

Comment:
What permissions does it get again?

Also should we say it also creates a namespace scoped service account if it does not exist?

Reply:
admin, by default. i kept it vague so that they click on the namespace scoped service accounts to learn more.